src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/ChimericAlignment.java

```diff
@@ -161,10 +160,10 @@ public ChimericAlignment(final AlignmentInterval intervalWithLowerCoordOnContig,
                 }
             }
 
-            final boolean isNotSimpleTranslocation = isNotSimpleTranslocation(current, next,
-                    determineStrandSwitch(current, next), involvesRefPositionSwitch(current, next));
-            if (isNotSimpleTranslocation)
-                results.add(new ChimericAlignment(current, next, insertionMappings, alignedContig.contigName));
+            final ChimericAlignment chimericAlignment = new ChimericAlignment(current, next, insertionMappings, alignedContig.contigName);
```

Comment:
method to be phased out, but it makes sense to let down stream users to check what type of novel adjacency this CA could point to, which requests the state checkers to be more comprehensive